Snowdog_Components/components/Organisms/order-summary/_order-summary-variables.scss

```diff
@@ -72,6 +72,3 @@ $order-summary__product-qty-color                   : $gray-dark !default;
 $order-summary__product-price-font-size             : $font-size-base !default;
 $order-summary__product-wrapper-width               : 100% !default;
```

Comment:
after removing the new variables there's some unnecessary space between variable names and `:`